README.md

```diff
@@ -1,5 +1,5 @@
 # fcc-landing-product-page-apple.markdown
-<h2> Develop Branch </h2>
+<h3> Feature Branch </h3>
```

Comment:
when you use tags you don't have to add a space between the` <h3>` and Feature
so: `<h3>Feature Branch</h3>` suffices

Reply:
Hey Alana!

Sorry for the delay in response. Not sure why github didn't notify me that you made a comment.. But got it! 👍 I'll keep that in mind next time!